app/models/concerns/resource_tools/json.rb

```diff
@@ -1,8 +1,6 @@
 # frozen_string_literal: true
 
 # Handle JSON parsing.
-# Currently uses Hashie, but we can probably eliminate that for something
-# more lightweight
 module ResourceTools::Json
```

Comment:
All the behaviour we needed was built into `ActiveSupport::HashWithIndifferentAccess` so migrated to that instead.

Reply:
Where does this class get used? Can't see much from searching

Reply:
It gets automatically subclassed for each resource  via the `json` method, creating for instance `Event::JsonHandler`. (Actually, *only* Event::JsonHandler in the case of the event warehouse, lots of classes in the other warehouses)

The json method also returns this class, so:
```
    def create_or_update_from_json(json_data, lims)
      create_or_update(json.collection_from(json_data, lims))
    end
```
This bit is where its actually used. Essentially its just to perform the translations, as well as stringifying keys (I think the ML warehouse does a little more in this section).

On explaining it all here, `json` would probably be better named `json_handler`

Reply:
I'll add a bit of documentation here, as its probably quite useful. Might see if I can rename the method as well

Reply:
Documentation added